keps/sig-node/0035-20190130-topology-manager.md

```diff
@@ -176,72 +180,90 @@ of each pod in order to prioritize the importance of fulfilling optimal locality
+#### Topology Policy in Pod Spec
+The value of `v1core.PodSpec.Topology.Policy` field indicates the desired topology of Pod's resource assignment.
+This value determines how Topology Manager coordinates hardware resource assignment of container during Pod admission.
+If the value is omitted or equal to `none`, the Topology Manager will not intervene in Pod admission.
 
-**best-effort**: Topology Manager will provide the preferred allocation for the container based
-on the hints provided by the Hint Providers. If an undesirable allocation is determined, the pod will be admitted with this undesirable allocation.
-
-**restricted**: Topology Manager will provide the preferred allocation for the container based
-on the hints provided by the Hint Providers. If an undesirable allocation is determined, the pod will be rejected. 
-This will result in the pod being in a `Terminated` state, with a pod admission failure.
-
-**single-numa-node**: Topology mananager will enforce an allocation of all resources on a single NUMA Node. If such
-an allocation is not possible, the pod will be rejected. This will result in the pod being in a `Terminated` state, with a pod admission failure.
+```
+apiVersion: v1
+kind: Pod
+Spec:
+  topology:
+    policy: restricted
+  containers:
+  - name: cnn
+    Resources:
+      Requests:
+        cpu: 4
+        memory: 5Gbi
+        nvidia.com/gpu: 4
+      Limits:
+        cpu: 4
+        memory: 5Gbi
+        nvidia.com/gpu: 4
+```
+_Listing: Extended pod spec with a topology field._
 
-The Topology Manager component will be disabled behind a feature gate until
-graduation from alpha to beta.
+The behavior of Topology Manager for each policy is described in the below section:
+- **none (default)**: Kubelet does not consult Topology Manager for placement decisions.
+- **best-effort**: Topology Manager will provide the preferred allocation for the container based on the hints provided by the Hint Providers. If an undesirable allocation is determined, the pod will be admitted with this undesirable allocation.
+- **restricted**: Topology Manager will provide the preferred allocation for the container based on the hints provided by the Hint Providers. If an undesirable allocation is determined, the pod will be rejected. This will result in the pod being in a `Terminated` state, with a pod admission failure.
+- **single-numa-node**: Topology manager will enforce an allocation of all resources on a single NUMA Node. If such an allocation is not possible, the pod will be rejected. This will result in the pod being in a `Terminated` state, with a pod admission failure.
+- **pod-level-single-numa-node**: Topology manager will enforce an allocation of all containers in the pod (thus their resources) on the same NUMA Node. If such an allocation is not possible, the pod will be rejected. This will result in the pod being in a `Terminated` state, with a pod admission failure.
```

Comment:
It was hard to distinguish the difference between **single-numa-node** and **pod-level-single-numa-node**.
The first sentence is important, I tried to strongly point out the difference.